swagger.json

```diff
@@ -6855,15 +6855,30 @@
                             "items": {
                                 "type": "object",
                                 "properties": {
-                                    "tagName": {
+                                    "tagId": {
+                                        "type": "number",
+                                        "format": "int64",
+                                        "description": "The id of the tag to grant this user access to.",
+                                        "example": "123"
+                                    },
+                                    "roleName" : {
                                         "type": "string",
-                                        "description": "The name of the tag to grant this user access to.",
-                                        "example": "North Carolina Distribution"
+                                        "description": "The name of the standard role type (case insensitive). Either this field or customRoleUuid must be provided.",
+                                        "example": "Read-Only Admin",
+                                        "enum": [
+                                            "Full Admin",
+                                            "Read-only Admin",
+                                            "Dispatch",
+                                            "Maintenance",
+                                            "Full Admin (No Dash Cam Access)",
+                                            "Read-only Admin (No Dash Cam Access)",
+                                            "Maintenance"
+                                        ]
                                     },
-                                    "role" : {
+                                    "customRoleUuid" : {
                                         "type": "string",
-                                        "description": "The name of the role the user is assigned to for this tag. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
-                                        "example": "Read-Only Admin"
+                                        "description": "The uuid of the custom role the user is assigned to for this tag. Either this field or roleId must be provided.",
+                                        "example": "8a9371af-82d1-4158-bf91-4ecc8d3a114c"
```

Comment:
Are we tracking the addition of custom role `/GET` in another ticket?

Reply:
added a get endpoint for `/user_roles`